src/box/lua/config/cluster_config.lua

```diff
@@ -316,10 +341,17 @@ return schema.new('cluster_config', record_from_fields({
             -- are going to be merged into the main config
             -- (i.e. the validation is performed after the version
             -- comparison).
-            key = schema.scalar({type = 'string'}),
-            value = schema.scalar({type = 'any'}),
+            key = schema.scalar({
+                type = 'string',
+                description = descriptions['conditional.*']
+            }),
```

Comment:
The same remark here: I don't think that descriptions on keys are really useful. See [here](https://github.com/tarantool/tarantool/pull/10629#discussion_r1794184101).

Reply:
Done.